src/PowerupSubsystem.cpp

```diff
@@ -12,7 +12,9 @@ namespace {
     const float POWERUP_DISTANCE_THRESHOLD = 2.5f;
     const int POWERUP_LOCK_FRAMES = 30;
     const glm::vec3 POWERUP_LOCATION_LIMITS = glm::vec3(10.0f, 1.5f, 10.0f);
-
+    glm::vec3 charcoal_locations[20];
```

Comment:
The 20 is pretty fragile, for a number that we will probably play with in the future.